step.yml

```diff
@@ -1,9 +1,14 @@
 title: Step readme generator
-summary: Generates a standard README.md based on step.yml data
-description: Generates a standard README.md based on step.yml data
-website: https://github.com/bitrise-steplib/bitrise-step-step-readme-generator
-source_code_url: https://github.com/bitrise-steplib/bitrise-step-step-readme-generator
-support_url: https://github.com/bitrise-steplib/bitrise-step-step-readme-generator/issues
+summary: Generate a `README.md` based on `step.yml` data
+description: |-
+  This step can be used to generate a standard README.md based on step.yml data. The resulting readme will list the step's most important details, such as the description, a table of inputs and outputs, and links for further information.
+
+  The exact structure is defined in `README.md.gotemplate`
+
+  Additional step-specific content can be included from local Markdown files, such as example usage and information for contributors.
+website: https://github.com/bitrise-steplib/steps-readme-generator
+source_code_url: https://github.com/bitrise-steplib/steps-readme-generator
+support_url: https://github.com/bitrise-steplib/steps-readme-generator/issues
```

Comment:
NOTE: A project_type is needed afaik for unified CI, so it can select Stacks.

Reply:
Looks like it works fine without a `project_type`:
https://app.bitrise.io/build/4c4d937d-5e8f-4b89-b721-bf90fbab0ec8#?tab=log

Reply:
According to [Devcenter](https://devcenter.bitrise.io/contributors/create-your-own-step/#platform-types), it's best to leave undefined if the step applies to all platforms:

> The available platform types are controlled by the project_type_tags attribute. If your Step is available for every platform or project type, do not specify project_type_tags.

Reply:
That's not internally consistent as it also says:
> Use utility only if you believe none of the other types fit your Step.

Reply:
But that's `type_tags`, not `project_type_tags`. I didn't define `type_tags` because this step won't be published to the step library, so it won't appear on the integrations page where `type_tags` is used as a filter